roles/openshift_master/defaults/main.yml

```diff
@@ -29,4 +29,4 @@ os_firewall_deny:
 - service: former etcd peer port
   port: 7001/tcp
 
-openshift_version: "{{ openshift_pkg_version | default(openshift_image_tag) | default(openshift.common.image_tag) | default('') }}"
+openshift_version: "{{ openshift_pkg_version | default(openshift_image_tag) | default(openshift.docker.openshift_image_tag) | default('') }}"
```

Comment:
ok, I'll have to try that out.  I feel like I tested the current defaulting but I can't say I know what default is supposed to do if you pass it something that isn't defined.